WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -17,6 +17,8 @@ struct CartView: View {
             DynamicHStack(spacing: Constants.cartHeaderSpacing) {
                 HStack {
                     backAddMoreButton
+                        .padding(.top, Constants.headerPadding)
+                        .padding(.trailing, Constants.headerPadding)
```

Comment:
Seems odd to use the same constant for the trailing here – it's a coincidence that they need the same value. It might be better to use `spacing` on the HStack to achieve this one.

Reply:
Oh, that's definitely better. Updated here: https://github.com/woocommerce/woocommerce-ios/pull/13707/commits/8af4878320f0de74e2849841bbfe8dddf7db62aa